src/components/containers/MenuPanel.js

```diff
@@ -11,20 +12,15 @@ export default class MenuPanel extends Component {
     this.togglePanel = this.togglePanel.bind(this);
   }
 
-  menupanelClasses() {
-    if (this.props.iconClass) {
-      return {
-        iconClass: `menupanel__icon ${this.props.iconClass}`,
-        spanClass: 'menupanel__icon-span',
-      };
-    } else if (this.props.question) {
+  getIcon() {
+    if (this.props.question) {
       return {
-        iconClass: 'menupanel__icon icon-question-circle',
+        iconType: <QuestionIcon className="menupanel__icon" />,
```

Comment:
This could just be named `icon` as `iconType` makes me think that it will be a string describing the type of icon.